src/nvhttp.cpp

```diff
@@ -267,12 +268,32 @@ namespace nvhttp {
 
     launch_session.host_audio = host_audio;
     launch_session.gcm_key = util::from_hex<crypto::aes_t>(get_arg(args, "rikey"), true);
+    std::stringstream mode = std::stringstream(get_arg(args,"mode"));
+    //Split mode by the char "x", to populate width/height/fps
+    int x = 0;
+    std::string segment;
+    while(std::getline(mode, segment, 'x'))
+    {
+      if(x == 0)launch_session.width = stoi(segment);
+      if(x == 1)launch_session.height = stoi(segment);
+      if(x == 2)launch_session.fps = stoi(segment);
+      x++;
+    }
+    launch_session.unique_id = (get_arg(args,"uniqueid"));
+    launch_session.uuid = (get_arg(args,"uuid"));
+    launch_session.appid = util::from_view(get_arg(args,"appid"));
+    launch_session.enable_sops = util::from_view(get_arg(args,"sops"));
+    launch_session.gcmap = util::from_view(get_arg(args,"gcmap"));
+    launch_session.enable_hdr = 0;
+    if(args.find("enableHdr"s) != std::end(args)){
+      launch_session.enable_hdr = util::from_view(get_arg(args,"enableHdr"));
+    }
```

Comment:
Should we also expose the audio configuration (maybe in a friendly form like `2.0`/`5.1`/7.1`)?

Reply:
Since the system uses the parameters passed by the "/launch" endpoint. I have a `surroundAudioInfo` param. How should the parameter be interpreted?

Reply:
The bottom 16-bits is a channel count, so we could just expose that. 

https://github.com/moonlight-stream/moonlight-common-c/blob/49fef0383020272941c4704d2ecf3e1e71174092/src/Limelight.h#L214-L217